community-tutorials/keyhelp/01-en.md

```diff
@@ -0,0 +1,100 @@
+---
+title: How to install Keyhelp
+description: How to install Keyhelp on a Netcup VM
+updated_at: 2021-11-04
+slug: install-keyhelp
+author_name: Andreas Fink
+author_url: https://andreasf.ink
+author_image: -
+author_bio: 
+tags: [keyhelp, webhost, mailserver]
+netcup_product_url: https://www.netcup.de/bestellen/produkt.php?produkt=2554
+language: en
+available_languages: en
+---
+
+# Introduction
+This tutorial will show you how to install KeyHelp on a Netcup `VPS 1000 G9`.
+
+# Requirements
+You need an empty VPS with root access with the OS Debian or Ubuntu. I already had installations with only 1GB RAM working just fine, but I would recommend a minimum of 4GB RAM and 2 cores, just like a `VPS 500 G8`.
+A Domain pointed to your VPS that you will use for the controlpanel. In this tutorial I will use `example.com`.
+
+# Step 1 - Dependencies
+First, update your package manager.
+
+```
+apt update
+```
+
+Please install the packages `wget` and `ca-certificates`, so that no error occures during the installation.
+
+```
+apt install wget ca-certificates -y
+```
+
+# Step 2 - Install KeyHelp
+You can install KeyHelp just by pasting this line:
+```
+wget https://install.keyhelp.de/get_keyhelp.php -O install_keyhelp.sh ; bash install_keyhelp.sh ;
+```
+and pressing <kbd>Enter</kbd>.
+
+The installer will be downloaded and executes itself automatically.
+
+At this point, PHP, Keyhelp and IonCube will be installed.
+
+## Step 2.1 - Language
+![Languagesettings](./images/lang.png)
+Now, you will be prompted with this. If you want to keep the installation in english, just continue by entering `N` and pressing <kbd>Enter</kbd>. You can exit the installer anytime by entering `Q`.
+
+If you want to continue in another language, enter `L` and press <kbd>Enter</kbd>
+
+![Languages](./images/languages.png)
+Now, you can choose what language you want, type in the countrycode next to your language in brackets and press <kbd>Enter</kbd>.
+
+Now continue by entering `N` and pressing <kbd>Enter</kbd>.
+
+## Step 2.2 - Host settings
+![Settings default](./images/settings_0.png)
+You can now customize installation parameters. For me, I change the the hostname to `example.com` by entering `1` for editing the hostname and pressing <kbd>Enter</kbd>. I then enter the hostname `web.domain.com` and confirm it by pressing <kbd>Enter</kbd>. I now repeat this for the other settings.

```

Comment:
which domain are you switching to now? To example.com or to web.domain.com?

Reply:
@raphi-0901 Ah, forgot this one. Fixed now and replaced also the screenshot.